test/test_mathics_precedence.py

```diff
@@ -0,0 +1,606 @@
+    "Equivalent",
+    "Nor",
+    "Or",
+    "Xor",
+    "And",
+    "Nand",
+    "Not",
+    "NotReverseElement",
+    "NotSquareSubsetEqual",
+    "NotSquareSupersetEqual",
+    "NotSubset",
+    "NotSubsetEqual",
+    "NotSuperset",
+    "NotSupersetEqual",
+    "ReverseElement",
+    "SquareSubset",
+    "SquareSubsetEqual",
+    "SquareSuperset",
+    "SquareSupersetEqual",
+    "Subset",
+    "SubsetEqual",
+    "Superset",
+    "SupersetEqual",
+    "DoubleLeftArrow",
+    "DoubleLeftRightArrow",
+    "DoubleRightArrow",
+    "DownLeftRightVector",
+    "DownLeftTeeVector",
+    "DownLeftVector",
+    "DownLeftVectorBar",
+    "DownRightTeeVector",
+    "DownRightVector",
+    "DownRightVectorBar",
+    "LeftArrow",
+    "LeftArrowBar",
+    "LeftArrowRightArrow",
+    "LeftRightArrow",
+    "LeftRightVector",
+    "LeftTeeArrow",
+    "LeftTeeVector",
+    "LeftVector",
+    "LeftVectorBar",
+    "LowerLeftArrow",
+    "LowerRightArrow",
+    "RightArrow",
+    "RightArrowBar",
+    "RightArrowLeftArrow",
+    "RightTeeArrow",
+    "RightTeeVector",
+    "RightVector",
+    "RightVectorBar",
+    "ShortLeftArrow",
+    "ShortRightArrow",
+    "UpperLeftArrow",
+    "UpperRightArrow",
+    "DoubleVerticalBar",
+    "NotDoubleVerticalBar",
+    "VerticalBar",
+    "Equal",
+    "Greater",
+    "GreaterEqual",
+    "Less",
+    "LessEqual",
+    "GreaterSlantEqual",
+    "LessSlantEqual",
+    "SameQ",
+    "Unequal",
+    "UnsameQ",
+    "Congruent",
+    "CupCap",
+    "DotEqual",
+    "EqualTilde",
+    "Equilibrium",
+    "GreaterEqualLess",
+    "GreaterFullEqual",
+    "GreaterGreater",
+    "GreaterLess",
+    "GreaterTilde",
+    "HumpDownHump",
+    "HumpEqual",
+    "LeftTriangle",
+    "LeftTriangleBar",
+    "LeftTriangleEqual",
+    "LessEqualGreater",
+    "LessFullEqual",
+    "LessGreater",
+    "LessLess",
+    "LessTilde",
+    "NestedGreaterGreater",
+    "NestedLessLess",
+    "NotCongruent",
+    "NotCupCap",
+    "NotGreater",
+    "NotGreaterEqual",
+    "NotGreaterFullEqual",
+    "NotGreaterLess",
+    "NotGreaterTilde",
+    "NotLeftTriangle",
+    "NotLeftTriangleEqual",
+    "NotLess",
+    "NotLessEqual",
+    "NotLessFullEqual",
+    "NotLessGreater",
+    "NotLessTilde",
+    "NotPrecedes",
+    "NotPrecedesSlantEqual",
+    "NotPrecedesTilde",
+    "NotRightTriangle",
+    "NotRightTriangleEqual",
+    "NotSucceeds",
+    "NotSucceedsSlantEqual",
+    "NotSucceedsTilde",
+    "NotTilde",
+    "NotTildeEqual",
+    "NotTildeFullEqual",
+    "NotTildeTilde",
+    "Precedes",
+    "PrecedesEqual",
+    "PrecedesSlantEqual",
+    "PrecedesTilde",
+    "Proportion",
+    "Proportional",
+    "ReverseEquilibrium",
+    "RightTriangle",
+    "RightTriangleBar",
+    "RightTriangleEqual",
+    "Succeeds",
+    "SucceedsEqual",
+    "SucceedsSlantEqual",
+    "SucceedsTilde",
+    "Tilde",
+    "TildeEqual",
+    "TildeFullEqual",
+    "TildeTilde",
+    # In Mathics, the precedence of these operators is quite low.
```

Comment:
Is there a reason to chose these lower values for the precedence of these operators?

Reply:
> Is there a reason to choose these lower values for the precedence of these operators?

See https://github.com/Mathics3/mathics-scanner/pull/99#discussion_r1863620711